stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -64,11 +64,25 @@ spec:
             echo "Waiting for Vault to come up...";
             {{- if .Values.vault.enabled }}
             until nc -z -w 2 {{ include "pipelines.vault.name" . }} {{ .Values.vault.service.port }} && echo Vault ok; do
+              sleep 2;
+            done;
             {{- else }}
-            until nc -z -w 2 {{ tpl .Values.global.vault.host . }} {{ .Values.global.vault.port }} && echo Vault ok; do
-            {{- end }}
+            is_vault_up=0;
+            initilized_check_endpoint="/v1/sys/init";
+            until [ "$is_vault_up" == "1" ] && echo Vault ok; do
```

Comment:
Use two square brackets here. It is also better to use two instead of one.

Reply:
Used two square brackets